pyg_lib/csrc/sampler/random_walk.cpp

```diff
@@ -0,0 +1,28 @@
+#include "random_walk.h"
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+
+namespace pyg {
+namespace sampler {
+
+at::Tensor random_walk(const at::Tensor& rowptr,
+                       const at::Tensor& col,
+                       const at::Tensor& seed,
+                       int64_t walk_length,
+                       double p,
+                       double q) {
+  static auto op = c10::Dispatcher::singleton()
```

Comment:
Is this intended or you just want to make this code work end to end?
I was thinking that you are going to at least copy the code you write in a different repo to this one.

Reply:
It's still WIP :)